parm/config/gfs/config.prepoceanobs

```diff
@@ -7,15 +7,20 @@ echo "BEGIN: config.prepoceanobs"
 export OCNOBS2IODAEXEC=${HOMEgfs}/sorc/gdas.cd/build/bin/gdas_obsprovider2ioda.x
 
 export SOCA_INPUT_FIX_DIR=@SOCA_INPUT_FIX_DIR@
-export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/soca/obs/config
+
+export OBS_YAML_DIR=${OBS_YAML_DIR:-${PARMgfs}/gdas/soca/obs/config}
 export OBSPREP_YAML=@OBSPREP_YAML@
 export OBS_LIST=@SOCA_OBS_LIST@
-[[ -n "${OBS_LIST}" ]] || export OBS_LIST=${HOMEgfs}/sorc/gdas.cd/parm/soca/obs/obs_list.yaml
+export OBS_LIST=${OBS_LIST:-${PARMgfs}/gdas/soca/obs/obs_list.yaml}
 export OBS_YAML=${OBS_LIST}
 
 # ocean analysis needs own dmpdir until standard dmpdir has full ocean obs
 export DMPDIR=@DMPDIR@
 
+# For BUFR2IODA json and python scripts
+export JSON_TMPL_DIR=${JSON_TMPL_DIR:-"${PARMgfs}/gdas/ioda/bufr2ioda"}
+export BUFR2IODA_PY_DIR=${BUFR2IODA_PY_DIR:-"${USHgfs}"}
```

Comment:
```suggestion
export JSON_TMPL_DIR="${PARMgfs}/gdas/ioda/bufr2ioda"
export BUFR2IODA_PY_DIR="${USHgfs}"
```
There is no reference to `JSON_TMPL_DIR` or `BUFR2IODA_PY` before this point and there shouldn't be.  This is the first time it will be defined.

Reply:
This is to accommodate GDASApp ctests, which don't run the link script

Reply:
changed